wsf_scraping/spiders/base_spider.py

```diff
@@ -29,3 +32,17 @@ def _check_headers(self, response_headers,
                        desired_extension=b'application/pdf'):
         content_type = response_headers.get('content-type', '').split(b';')[0]
         return desired_extension == content_type
+
+    def _save_file(self, url, response_body):
+        filename = os.path.basename(urlparse(url).path)
```

Comment:
much better than the hacky `filename = urlparse(response.url).path.split('/')[-1]`

Reply:
😇